src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AFSamplers.java

```diff
@@ -0,0 +1,112 @@
+package org.broadinstitute.hellbender.tools.exome.allelefraction;
+
+import org.apache.commons.math3.random.RandomGenerator;
+import org.broadinstitute.hellbender.utils.mcmc.AdaptiveMetropolisSampler;
+import org.broadinstitute.hellbender.utils.mcmc.Sampler;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by davidben on 11/25/15.
+ */
+public class AFSamplers {
+
+    public static final class MeanBiasSampler extends AdaptiveMetropolisSampler<AFState, AFData> {
```

Comment:
For each of these Samplers, can you write a javadoc explaining the variable that it samples (using the nomenclature of the model pdf)?